packages/dds/tree/src/feature-libraries/modular-schema/comparison.ts

```diff
@@ -65,27 +65,15 @@ export function allowsTreeSuperset(
 
 	if (original instanceof MapNodeStoredSchema) {
 		if (superset instanceof MapNodeStoredSchema) {
-			return allowsFieldSuperset(
-				policy,
-				originalData,
-				normalizeField(original.mapFields),
```

Comment:
all normalizeField did was handle undefined, which the type system says cannot occur here so it's useless and thus removed.